warmup.krun

```diff
@@ -124,16 +123,16 @@ SKIP = [
     #
     #"*:PyPy:*",
     "*:CPython:*",
-    #"*:Hotspot:*",
+    #"*:HotSpot:*",
     #"*:Graal:*",
     #"*:LuaJIT:*",
     #"*:HHVM:*",
-    #"*:JRubyTruffle:*",
+    #"*:TruffleRuby:*",
     #"*:V8:*",
     #"*:C:*",
 
     # This benchmark is slow *and* crashes
-    "fasta:JRubyTruffle:default-ruby",
+    "fasta:TruffleRuby:default-ruby",
```

Comment:
These two benchmarks are still too slow to run. One would add a week, the other a month. Going to look for others that might do the same now.